### PR TITLE
Apply paper-dialog-behavior instead of nesting paper-dialogs

### PR DESCRIPTION
**Pull request by NovapaX**
Paving the way for uniform behavior and styling of dialogs.

### CODE REVIEW

File: src/dialogs/ha-more-info-dialog.html [hunk 1]
```diff
@@ -33,81 +33,72 @@
...
   </template>
 </dom-module>
 
 <script>
-class HaMoreInfoDialog extends window.hassMixins.EventsMixin(Polymer.Element) {
+class HaMoreInfoDialog extends window.hassMixins.DialogMixin(Polymer.Element) {
   static get is() { return 'ha-more-info-dialog'; }
   static get properties() {
     return {
-      hass: Object,
```
Comment (andrey-git) [inline src/dialogs/ha-more-info-dialog.html]: You still need hass

Comment (NovapaX) [reply src/dialogs/ha-more-info-dialog.html]: It's part of the mixin. 
But if that's a bit to DRY I'm happy to change that back.

Comment (andrey-git) [reply src/dialogs/ha-more-info-dialog.html]: I now see you have put it in the mixin. I think it should be in the class directly, as the mixin doesn't interface with `hass` in any way.

**Review by andrey-git — APPROVED** [review]

File: src/dialogs/ha-more-info-dialog.html (hunk 1 above)
Comment (NovapaX) [reply src/dialogs/ha-more-info-dialog.html]: yes, agree. Kind of weird to have the observers on hass in the class, without the hass property tienelf. Changing.

**Pull request merged**